services/city.service.go

```diff
@@ -0,0 +1,28 @@
+package services
+
+import (
+	"fmt"
+	"project_weather/generated/dao/model"
+)
+
+type CityService struct{}
+
+func NewCityService() *CityService {
+	return &CityService{}
+}
+
+func (s *CityService) FormatWeatherDataForCity(weatherData map[string]interface{}) *model.City {
+	name := weatherData["city"].(map[string]interface{})["name"].(string)
```

Comment:
It will be great to create a DTO (https://en.wikipedia.org/wiki/Data_transfer_object) that represents the `weatherData` and is compatible with the `encoding/json` library in Golang.